FormalConjectures/Other/EquationalTheories_677_255.lean

```diff
@@ -35,8 +35,8 @@ abbrev Equation677 (G: Type) [Magma G] := ∀ x y : G, x = y ◇ (x ◇ ((y ◇
 
 @[category research solved, AMS 8]
 theorem Equation255_not_implies_Equation677 :
-    ∃ (G : Type) (_ : Magma G), Equation255 G ∧ ¬ Equation677 G := by
-  sorry
+    ∃ (G : Type) (_ : Magma G), Equation255 G ∧ ¬ Equation677 G :=
+  ⟨Fin 3, ⟨![![1, 2, 0], ![2, 0, 1], ![0, 1, 2]]⟩, fun x ↦ by fin_cases x <;> rfl, of_decide_eq_false rfl⟩
```

Comment:
```suggestion
    ∃ (G : Type) (_ : Magma G), Equation255 G ∧ ¬ Equation677 G :=
  ⟨Fin 3, ⟨![![1, 2, 0], ![2, 0, 1], ![0, 1, 2]]⟩, 
   fun x ↦ by fin_cases x <;> rfl, of_decide_eq_false rfl⟩
```

Just eyeballing this: is this over the 100 characters? If yes, let's split up the line..